controllers/inverse_kinematics_pyikfast/inverse_kinematics_pyikfast.py

```diff
@@ -68,9 +68,9 @@
 maxPositions = []
 for i in range(n):
     device = supervisor.getDeviceByIndex(i)
-    print(device.getName(), '   - NodeType:', device.getNodeType())
+    print(device.getName(), "   - NodeType:", device.getNodeType())
```

Comment:
Why change to double quote?

Reply:
this is auto formatting by black (https://github.com/psf/black) sorry for the distractions it caused